src/tutorials/instance_internet.md

```diff
@@ -0,0 +1,149 @@
+# Exposing an Instance to the Internet
+
+In this tutorial, you will learn how to expose a Caido instance to the internet.
+
+::: warning NOTE
+Ensure to replace `user` with your username, `example.com` with your domain, `user@example.com` with your email address, and account for any currently running processes by changing the ports.
+:::
+
+## Nginx Configuration
+
+1. To logically separate the internet-exposed Caido instance from your existing setup, create a new subdomain by adding a A record for `caido.example.com` for the IP address of your server.
+
+2. SSH into your server.
+
+3. Create a new `sites-available` file and use the `proxy_pass` directive to route traffic to Caido: `sudo nano /etc/nginx/sites-available/caido.example.com`
+
+```txt
+server {
+    server_name caido.example.com;
+    
+    location / {
+        proxy_pass http://127.0.0.1:8081;
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+    }
+    
+    listen 80;
+    listen [::]:80;
+}
+```
+
+4. Make the site available, test the configuration, and reload the web server:
+
+```bash
+sudo ln -s /etc/nginx/sites-available/caido.example.com /etc/nginx/sites-enabled/
+```
+
+```bash
+sudo nginx -t
+```
+
+```bash
+sudo systemctl reload nginx
+```
+
+5. Obtain a SSL/TLS certificate:
+
+```bash
+sudo certbot --nginx -d caido.example.com
+```
+
+6. Launch the Caido CLI:
+
+```bash
+caido-cli --ui-listen 0.0.0.0:8081 --proxy-listen 0.0.0.0:8082 --ui-domain caido.example.com --debug --no-renderer-sandbox --no-open
```

Comment:
Doesn't this defeat the purpose of the nginx server since you are listening on `0.0.0.0`?

I believe this should be `--ui-listen 127.0.0.1:8081 --proxy-listen 127.0.0.1:8082`